source/tutorials/quickstart-flakes.md

```diff
@@ -0,0 +1,146 @@
+# Quickstart Guide to Flakes
+
+This guide focuses on things that you need to know as a beginner,
+directing you where to go for more information on each particular topic.
+
+## Templates
+
+You don't have to write flakes from scratch, there are several
+templates you can use.
+To see the available templates:
+
+```
+nix flake show templates
+```
+
+To create a flake from a specific template:
+
+```
+nix flake init -t templates#simpleContainer
+```
+
+## Flake structure
+
+The basic structure of a flake is
+
+```
+{
+  description = ... # package description
+  inputs = ... # dependencies
+  outputs = ... # what the flake produces
+  nixConfig = ... # advanced configuration options
+}
+```
+
+The `description` part is self-explanatory.
+You probably won't need `nixConfig` unless you're doing something fancy.
+We'll focus on what goes into the `inputs` and `outputs` sections,
+and highlight some of the things that beginners may find confusing.
+
+## Inputs
+
+This section specifies the dependencies of a flake.
+Each entry maps an input name to a *flake reference*.
+The most common type of flake reference is a
+Git or Mercurial repositories.
+Here's a simple example:
+
+```
+inputs.import-cargo = {
+  type = "github";
+  owner = "edolstra";
+  repo = "import-cargo";
+};
+```
+
+Equivalently, you could just write:
```

Comment:
The manual still needs a name for the "URL-like syntax", but let's just assume that's what it's called:

```suggestion
Equivalently, you can use the URL-like shorthand syntax:
```